torchbenchmark/models/hf_Whisper/__init__.py

```diff
@@ -0,0 +1,19 @@
+from torchbenchmark.util.framework.huggingface.model_factory import HuggingFaceModel
+from torchbenchmark.tasks import SPEECH
+import torch
+
+class Model(HuggingFaceModel):
+    task = SPEECH.RECOGNITION
+    DEFAULT_EVAL_BSIZE = 8
+    
+    def __init__(self, test, device, jit=False, batch_size=None, extra_args=[]):
+        super().__init__(name="hf_Whisper", test=test, device=device, jit=jit, batch_size=batch_size, extra_args=extra_args)
+        self.feature_size = 80
+        self.sequence_length = 3000
+        self.input_features = torch.randn(size=(self.batch_size, self.feature_size, self.sequence_length),device=self.device).half()
```

Comment:
By default, we assume the input precision is FP32 (consistent with the default precision with pytorch). If the default input type precision is FP16, we need to implement a callback function `enable_fp16_half(self)`, convert the inputs to half() there, and set `DEFAULT_EVAL_CUDA_PRECISION = "fp16"` (https://github.com/pytorch/benchmark/blob/8177771c9d67c778a1f986fc8c7fc32008c5cfda/torchbenchmark/util/framework/timm/model_factory.py#L16C5-L16C41)